### PR TITLE
Refactors Panel, Titlebar and Window

### PR DESCRIPTION
**Pull request by KaiVolland**
### Description:

This is a breaking change as it removes some props from `Panel`, `Titlebar` and `Window`:

- Removes closable prop from all three classes
  - If you wan't to implement a close button just do it your self and pass it via the `tools` property.
- Removes collapsible prop from Titlebar
  - This prop was not used as the `Panel` passes a CollapseButton if you configure it with `collapsible`
- Removes unused parent property from Titlebar and Panel
- Renames `compress…` props to `collapse…`

It also changes the way the `Panel` component calculates its height and the height of its body.
The old defaults for the props `height` (100) and `width` (100) are replaced with `'auto'` so the Panel will grow with its content, except you specify a specific height and width.

@terrestris/devs plz review (and test)

### CODE REVIEW

File: src/Panel/Panel/Panel.jsx [hunk 1]
```diff
@@ -306,14 +349,12 @@ export class Panel extends React.Component {
     return (
       <Rnd
         className={rndClassName}
-        ref={(panel) => { this.panel = panel; }}
+        ref={(rnd) => { this.rnd = rnd; }}
         default={{
           x: isNumber(rndOpts.x) ? rndOpts.x : (window.innerWidth / 2) - 160,
           y: isNumber(rndOpts.y) ? rndOpts.y : (window.innerHeight / 2) - 100,
```
Comment (dnlkoch) [inline src/Panel/Panel/Panel.jsx]: Not related to this PR, but just curious where are these numbers (160 & 100) are from.

File: src/Panel/Panel/Panel.jsx [hunk 2]
```diff
@@ -306,14 +349,12 @@ export class Panel extends React.Component {
     return (
       <Rnd
         className={rndClassName}
-        ref={(panel) => { this.panel = panel; }}
+        ref={(rnd) => { this.rnd = rnd; }}
```
Comment (dnlkoch) [inline src/Panel/Panel/Panel.jsx]: Parentheses can be omitted.

**Comment by dnlkoch** [comment]
Great one! Only two minor non-blocking comments.

File: src/Panel/Panel/Panel.jsx (hunk 1 above)
Comment (KaiVolland) [reply src/Panel/Panel/Panel.jsx]: I don't know.

File: src/Panel/Panel/Panel.example.jsx [hunk 3]
```diff
@@ -12,23 +12,24 @@ render(
...
         x={200}
         y={20}
-        closable={true}
         collapsible={true}
-        title="Tooltips"
-        compressTooltip="Einklappen"
-        closeTooltip="Schließen"
-      />
+        title="Tooltip"
+        collapseTooltip="Einklappen"
```
Comment (ahennr) [inline src/Panel/Panel/Panel.example.jsx]: Maybe be could only use english texts in `react-geo`to make texts / tooltips more consistently.

Comment (KaiVolland) [reply src/Panel/Panel/Panel.example.jsx]: As this is the explicit example for setting alternate tooltips i used german.
The default value is ofcourse english: `collapse`

**Pull request merged**